src/core/session.ts

```diff
@@ -191,17 +191,17 @@ export class Session
     )
   }
 
-  followedLinkToLocation(link: Element, location: URL) {
+  followedLinkToLocation(link: HTMLAnchorElement, location: URL) {
```

Comment:
While I appreciate the type narrowing, so long as the `initiator` can invoke `dispatchEvent`, we might be able to achieve the outcome without disrupting these interfaces. 

I've been tempted to narrow this delegate method before, but I have a hunch that there is some Custom Element support reason that it's still `Element`.

Reply:
What would your advice be when updating the types here?:
https://github.com/hotwired/turbo/pull/695/files#diff-68b647dc2963716dc27c070f261d0b942ee9c00be7c4149ecb3a5acd94842d40R65

Reply:
https://github.com/hotwired/turbo/pull/695/commits/95590c442526fc27fd1542a7a8dc3d0478827bb8 uses `Element` for all references